pkg/tempo/dummyreceiver/factory.go

```diff
@@ -0,0 +1,47 @@
+package dummyreceiver
+
+import (
+	"context"
+
+	"go.opentelemetry.io/collector/component"
+	"go.opentelemetry.io/collector/config/configmodels"
+	"go.opentelemetry.io/collector/consumer"
+	"go.opentelemetry.io/collector/receiver/receiverhelper"
+)
+
+const (
+	// TypeStr for dummy receiver.
+	TypeStr = "dummy"
```

Comment:
What do you think of naming it `noop` instead?

Reply:
Yeah, I am good with this.